package Exchange;

message RemoteSubscription
{
  required int32 SubscriptionId=1;
}